ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -1481,6 +1492,54 @@ namespace OrdinalAxis {
             return ret;
         }
 
+        /**
+         * If overscroll is pixel or pecentage value, convert it to axis range.
+         *
+         * @private
+         * @param {number | string} overscroll
+         * Overscroll value in axis range, pixels or percentage value.
+         * @return {number}
+         * Overscroll value in axis range.
+         */
+        public convertOverscroll(overscroll : number | string = 0) : number {
+            const ordinal = this,
+                axis = ordinal.axis,
+                overscrollValue = parseInt(overscroll as any, 10);
+
+            if (!defined(axis.max) || !defined(axis.min)) {
+                return 0;
+            }
+
+            if (/%$/.test(overscroll as any)) {
+                // If overscroll is percentage
+                return (axis.max - axis.min) *
+                    overscrollValue / 100;
+            }
+
+            if (/px/.test(overscroll as any)) {
+                // If overscroll is pixels
+
+                // overscroll is limited to 90% of the axis width
+                // to prevent division by zero
+                const limitedOverscrollValue = Math.min(
+                        overscrollValue,
+                        axis.width * 0.9
```

Comment:
Use `axis.len`. In inverted chart width and height are swapped, but `axis.len` always refers to the length of the axis

Reply:
I'll do, thanks!